### PR TITLE
Add more HTML and top CSS LW

### PR DESCRIPTION
**Pull request by thetabloom**

### CODE REVIEW

**Review by thetabloom — COMMENTED** [review]
Adding styles to top of Idea Box. Still need to fix the size of the button issue.

File: index.html [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
+
   <body>
     <section class="input-section">
-      <h1>ideabox</h1>
+      <h1>idea<span>box</span></h1>
       <input class="input-title" placeholder="Title">
       <input class="input-body" placeholder="Body">
-      <button class="submitButton">
```
Comment (thetabloom) [inline index.html:13]: Changed name of submit-button to save-button

File: index.html [hunk 2]
```diff
@@ -24,6 +26,6 @@ <h3>Title</h3>
       <p class="idea-quality">Quality</p>
 
     </section>
-    <script type="text/javascript" src="index.js"></script>
```
Comment (thetabloom) [inline index.html:27]: revised java script scripting

File: styles.css [hunk 3]
```diff
@@ -1,2 +1,76 @@
-font-family: 'Roboto', sans-serif;
```
Comment (thetabloom) [inline styles.css:1]: Changed the Roboto to Roboto -Slab

File: index.html [hunk 4]
```diff
@@ -1,16 +1,18 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (thetabloom) [inline index.html:4]: shortened stylesheet to be able to add more styles than text

File: index.html [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto:700" rel="stylesheet">
+   
+    <link href="styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto+Slab:700" rel="stylesheet">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
+
   <body>
     <section class="input-section">
-      <h1>ideabox</h1>
```
Comment (thetabloom) [inline index.html:10]: added span for styling purposes

**Comment by saraebbers** [comment]
code looks great.  I'm having trouble with my branch, I may pull down yours from the master and try to do a new branch instead.

**Pull request merged**